src/services/utils.ts

```diff
@@ -0,0 +1,14 @@
+import { getPublicKey } from './connect';
+
+export const changeTitle = () => {
+    const input = <HTMLInputElement>document.querySelector('#secret-key-input');
+    const title = <HTMLInputElement>document.querySelector('#public-key-title');
```

Comment:
I don't think it's a very svelte-like way of doing things, you should make use of the reactivity by storing the public key, and also storing if there was an error or not, and have your component update automatically (i.e. react to those changes)